client/src/components/feedbacks/feedback.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+import { Container } from 'react-bootstrap';
+import axios from 'axios';
+import FeedbackSend from './send';
+import FeedbackComment from './comment';
+import '../../style/feedback.css';
+
+const id = require('shortid');
+
+class Feedback extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      allFedback: [],
+      pieceFedback: [],
+      maxCommentShow: 1,
+      start: 0,
+      last: 0,
+    };
+  }
+
+  componentDidMount() {
+    axios.get('http://localhost:8080/api/feedback/2')
```

Comment:
You can use singlton Api object (/client/src/services/api.js) to avoid to mix UI and fetching data. And for consistency